4337/certora/specs/ExecTransactionFromModule.spec

```diff
@@ -0,0 +1,60 @@
+using Account2 as safeContract;
+
+methods {
```

Comment:
I know that this is a WIP spec file, but I'd only include methods that are used by the rule here